cla_public/libs/mailgun.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from flask import current_app
+
+
+def send_ref_by_email(to, subject, email_body):
+    return requests.post(
+        'https://api.mailgun.net/v3/{domain}/messages'.format(
```

Comment:
Resolved off-line. Leaving as-is due to added complexity moving them to constants and referring other constants inside `format`.